Open-XML-SDK.sln

```diff
@@ -217,9 +217,9 @@ Global
 		{890B74DD-6316-4D56-B42A-5D66F10F88C6} = {C5AE39F0-A152-471A-B90E-B8F4E94AA6C2}
 		{5DE9FB8F-A7C2-4038-A4A8-1622CDB6821A} = {7DAF7304-40CC-4180-88A5-9A89DD13C565}
 		{F48B7D50-16CB-4BEF-A614-2DED9960AF09} = {7DAF7304-40CC-4180-88A5-9A89DD13C565}
-		{C91489AB-FF14-4FAD-BA51-35371ADD7E1C} = {7DAF7304-40CC-4180-88A5-9A89DD13C565}
-		{BB5DF535-E849-42AC-852A-A6D4815347C0} = {3653266D-2C88-4487-8977-839CB3E78A0A}
-		{5241BCF2-331D-428E-A9C4-F8DF92C8F726} = {A4DF60EB-3AA5-48F0-B4D2-3F94B8E62F03}
+		{C91489AB-FF14-4FAD-BA51-35371ADD7E1C} = {3653266D-2C88-4487-8977-839CB3E78A0A}
```

Comment:
the folder structure of the sln got messed up at some point. This just puts things back in the order we expect